examples/models/colors.py

```diff
@@ -1,3 +1,16 @@
+'''This example demonstrates the usage of grouping different shades
+of same color together using bokeh groups.
```

Comment:
```suggestion
of the same color together using Bokeh groups.
```
Please capitalize Bokeh (https://docs.bokeh.org/en/latest/docs/dev_guide/documentation.html#commonly-used-terms)